src/common/contracts/core/campaigns/client.ts

```
@@ -6,13 +6,21 @@ import { FULL_TGAS } from "@/common/constants";
+  CampaignDonation,
+  CampaignDonationArgs,
+  CampaignInputs,
+  CampaignsContractConfig,
+} from "./interfaces";
 
 const contractApi = naxiosInstance.contractApi({
   contractId: CAMPAIGNS_CONTRACT_ACCOUNT_ID,
   cache: new MemoryCache({ expirationTime: 10 }),
 });
 
+export const get_config = () => contractApi.view<{}, CampaignsContractConfig>("get_config");
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)